examples/travis/check-results.py

```diff
@@ -1,87 +1,109 @@
 from nbodykit.lab import *
-from nbodykit.cosmology import WMAP9, LinearPower
+import glob
 import os
-import numpy
+import os.path
+import numpy as np
+import scipy.interpolate
 from numpy.testing import assert_allclose
 
 from matplotlib.backends.backend_agg import FigureCanvasAgg
 from matplotlib.figure import Figure
 
-def compute_power(output, Nmesh=128):
+def compute_power(output):
     """Compute the compensated power spectrum from a catalogue."""
     catcdm = BigFileCatalog(output, dataset='1/', header='Header')
 
     try:
-        catb = MultipleSpeciesCatalog(["gas", "star"], 
+        catb = MultipleSpeciesCatalog(["gas", "star"],
                 BigFileCatalog(output, dataset='0/', header='Header'),
                 BigFileCatalog(output, dataset='4/', header='Header'))
     except:
         catb = BigFileCatalog(output, dataset='0/', header='Header')
+    box = catcdm.attrs['BoxSize']
+    Nmesh = 2*int(np.round(np.cbrt(catcdm.attrs['TotNumPart'][0])))
 
     meshcdm = catcdm.to_mesh(Nmesh=Nmesh, window='cic', compensated=True, interlaced=True)
     meshb = catb.to_mesh(Nmesh=Nmesh, window='cic', compensated=True, interlaced=True)
-    pkb = FFTPower(catb, mode='1d', Nmesh=128)
-    pkcdm = FFTPower(catcdm, mode='1d', Nmesh=128)
+    pkb = FFTPower(catb, mode='1d', Nmesh=Nmesh)
+    pkcdm = FFTPower(catcdm, mode='1d', Nmesh=Nmesh)
     z = 1. / catcdm.attrs['Time'] - 1
-    return pkcdm.power, pkb.power, z
+    box = catcdm.attrs['BoxSize']
+    omegab = catcdm.attrs['OmegaBaryon']
+    omega0 = catcdm.attrs['Omega0']
+    return pkcdm.power, pkb.power, z, box, omegab, omega0
 
-def test_power(output, refcdm, refb, ref_z, final=8):
+def test_power(output, transfer):
     """Check the initial power against linear theory and a linearly grown IC power"""
     print('testing', output)
-    pkcdm, pkb, z = compute_power(output)
+    pkcdm, pkb, z, box, omegab, omega0 = compute_power(output)
 
     #Check types have the same power
-    fig = Figure(figsize=(5, 5), dpi=200)
+    fig = Figure()
     canvas = FigureCanvasAgg(fig)
-    ax = fig.add_subplot(121)
+    ax = fig.add_subplot(111)
 
-    pklin = LinearPower(WMAP9, redshift=z)
+    ctf = os.path.join(transfer, "class_tk_99.dat")
+    if z < 98.5:
```

Comment:
why 98.5? Is there way way to notify z=99 is special here?

Reply:
Yes, that was hacky, sorry. I'll fix it.